src/utils/dictionaries/pages.json

```diff
@@ -0,0 +1 @@
s/online-bootcamp-programacion":"/us/coding-campus/online-coding-bootcamp","/es/coding-campus/bootcamp-programacion-panama":"/us/coding-campus/coding-bootcamp-panama","/es/coding-campus/bootcamp-programacion-ecuador":"/us/coding-campus/coding-bootcamp-ecuador","/es/coding-campus/bootcamp-programacion-santiago":"/us/coding-campus/coding-bootcamp-santiago","/us/apply":"/es/aplica","/us/awards":"/es/premios","/us/blog":"/es/blog-en-espanol","/us/next-dates":"/es/proximas-fechas","/es/categoria":"/us/cluster","/us/contact":"/es/contactanos","/us/cookies":"/es/cookie","/us/faq":"/es/preguntas-frecuentes","/us/financials":"/es/financiacion","/us/geekforce-career-support":"/es/geekforce","/us/geekpal-support":"/es/geekpal","/us/geeks-vs-others":"/es/geeks-vs-otros","/us/graduates":"/es/graduados","/us/index":"/es/inicio","/us/jobs":"/es/empleo","/us/outcomes":"/es/resultados","/us/partners":"/es/alianzas","/us/press":"/es/prensa","/us/pricing":"/es/precio","/us/privacy-policy":"/es/privacidad","/us/students-and-projects":"/es/alumnos-y-proyectos","/us/testimonials":"/es/testimonios","/us/terms-conditions":"/es/terminos","/us/thank-you":"/es/gracias","/us/the-academy":"/es/sobre-la-academia","/us/why-4geeks":"/es/porque-4geeks","/us/coding-bootcamps/full-time-full-stack-developer":"/es/coding-bootcamps/full-stack-full-time","/us/coding-bootcamps/part-time-full-stack-developer":"/es/coding-bootcamps/full-stack-part-time","/us/coding-bootcamps/machine-learning-engineering":"/es/coding-bootcamps/curso-inteligencia-artificial","/us/coding-bootcamps/software-engineer-bootcamp":"/es/coding-bootcamps/ingenieria-de-software-programacion","/us/coding-campus/coding-bootcamp-colombia":"/es/coding-campus/bootcamp-programacion-colombia","/us/coding-campus/coding-bootcamp-argentina-buenos-aires":"/es/coding-campus/bootcamp-programacion-argentina-buenos-aires","/us/coding-campus/coding-bootcamp-caracas":"/es/coding-campus/bootcamp-programacion-caracas","/us/coding-campus/coding-bootcamp-costa-rica":"/es/coding-campus/bootcamp-programacion-costa-rica","/us/coding-campus/coding-bootcamp-miami":"/es/coding-campus/bootcamp-programacion-miami","/us/coding-campus/coding-bootcamp-bolivia":"/es/coding-campus/bootcamp-programacion-bolivia","/us/coding-campus/coding-bootcamp-madrid":"/es/coding-campus/bootcamp-programacion-madrid","/us/coding-campus/coding-bootcamp-maracaibo":"/es/coding-campus/bootcamp-programacion-maracaibo","/us/coding-campus/coding-bootcamp-mexico":"/es/coding-campus/bootcamp-programacion-mexico","/us/coding-campus/coding-bootcamp-uruguay":"/es/coding-campus/bootcamp-programacion-uruguay","/us/coding-campus/online-coding-bootcamp":"/es/coding-campus/online-bootcamp-programacion","/us/coding-campus/coding-bootcamp-panama":"/es/coding-campus/bootcamp-programacion-panama","/us/coding-campus/coding-bootcamp-ecuador":"/es/coding-campus/bootcamp-programacion-ecuador","/us/coding-campus/coding-bootcamp-santiago":"/es/coding-campus/bootcamp-programacion-santiago"}]}
```

Comment:
> Changes applied to the original ~~`pathDictionary.json`~~
pages.json (all pages -> .yml and posts -> .md) 

I split the posts and pages dictionary section to avoid dictionary clutter